tests/src/Functional/DeveloperTest.php

```diff
@@ -301,4 +351,41 @@ protected function developerCreateByAdminTest() {
     $this->assertFalse($this->developerStorage->loadUnchanged($test_user['email']), 'Developer does not exists anymore.');
   }
 
+  /**
+   * Tests getCompanies() function of the developer entity.
+   *
+   * @see \Drupal\apigee_edge\Entity\Developer::getCompanies()
+   */
+  public function developerGetCompanyListTest() {
```

Comment:
Because you only use `$this->developer` and `$this->company` in this test method, maybe it would be better to use local variables instead. For example, `$this->company` is only not null when this test runs. You can remove these test entities from Apigee Edge in the end of this test method.